main.tf

```diff
@@ -0,0 +1,147 @@
+provider "aws" {
+  alias = "module"
+  region = "${var.provider_region}"
+  profile = "${var.profile}"
+  assume_role {
+    role_arn = "${var.cross_dev_ops_role_arn}"
```

Comment:
I think 'assume_role_arn' is a better variable name

Reply:
changed it to your suggestion.